docs/Installation.md

```diff
@@ -46,27 +46,53 @@ If your Python environment doesn't include `pip`, see these
 [instructions](https://packaging.python.org/guides/installing-using-linux-tools/#installing-pip-setuptools-wheel-with-linux-package-managers)
 on installing it.
 
-To install dependencies, go into the `python` subdirectory of the repository,
+To install dependencies, **go into the `python` subdirectory** of the repository,
 and run from the command line:
 
     pip3 install .
 
+## Setting up ML-Agent within Unity
+
+1. Open Unity and use it to open folder `ml-agents`/`unity-environment`. 
+2. Go to `Edit` -> `Project Settings` -> `Player`
+3. For each of the platforms you target 
+(**`PC, Mac and Linux Standalone`**, **`iOS`** or **`Android`**):
+    1. Go into `Other Settings`.
+    2. Select `Scripting Runtime Version` to 
+    `Experimental (.NET 4.6 Equivalent)`
+    3. In `Scripting Defined Symbols`, add the flag `ENABLE_TENSORFLOW`. 
+    After typing in, press Enter.
+4. Go to `File` -> `Save Project`
+5. Restart the Unity Editor.
+
+## Setting up TensorflowSharp Support
+
+[Download](https://s3.amazonaws.com/unity-ml-agents/0.3/TFSharpPlugin.unitypackage) TensorflowSharp plugin, and import it into Unity once downloaded by double clicking on it.  You can see if it was successfully imported by checking the TensorFlow files in the Project window under `Assets` -> `ML-Agents` -> `Plugins` -> `Computer`. 
+
+**Note**: If you don't see anything under `Assets`, drag the `ml-agents`/`unity-environment`/`Assets`/`ML-Agents` folder under `Assets` within Project window.
+
+## Play an example environment using pretrained model
+
+1. In the Project window, go to `Assets` -> `ML-Agents` -> `Examples` -> `3DBall` folder and open the `3DBall` scene file. 
+2. In the Hierarchy window, click on `Ball3DAcademy` -> `Ball3DBrain`. 
+3. In the Inspector window, under `Brain (Script)` -> `Brain Type`, change the `Brain Type` to `Internal`. 
+4. In the Project window, go to `Assets` -> `ML-Agents` -> `Examples` -> `3DBall` -> `TFModels` folder, drap the `3DBall` model file into Inspector window under `Brain (Script)` -> `Graph Model` if it is not already attached. 
+5. Click the `Play` button and you will see the platforms automatically adjusts itself using the pretrained model.
+
 ## Docker-based Installation
 
 If you'd like to use Docker for ML-Agents, please follow 
 [this guide](Using-Docker.md). 
 
-## Unity Packages
-
-You can download ML-Agents as Unity Packages:
+## Next Steps
 
- * [ML-Agents with TensorflowSharp Plugin](https://s3.amazonaws.com/unity-ml-agents/0.3/ML-AgentsWithPlugin.unitypackage)
- * [ML-Agents without TensorflowSharp Plugin](https://s3.amazonaws.com/unity-ml-agents/0.3/ML-AgentsNoPlugin.unitypackage)
- * [TensorflowSharp Plugin Only](https://s3.amazonaws.com/unity-ml-agents/0.3/TFSharpPlugin.unitypackage)
+* For an overview of ML-Agents go [here](ML-Agents-Overview.md).
```

Comment:
Why are we removing the packages links?

Reply:
Because 
1. People were confused with package to install before, only giving them one would make them understand easier. 
2. Even if they install the ML-Agents with TensorflowSharp plugin or without the plugin, they still need to clone the repo to get the python side of the code, so this package seems unnecessary. 
Vince, Arthur and I had a discussion about this several weeks ago.